### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by utay**

### CODE REVIEW

File: markdown/notifications.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+Format of a notification sent to a custom webhook endpoint.
+
+||| col |||
+
+Notification:
+
+```json
+{
+  "text" : "Text we send to you"
```
Comment (Soulou) [inline markdown/notifications.md]: That's definitely not enough, we've to send the full event wuth all the information 

```
{
  type: "deployment",
  data: {
    ...
  }
}
```

Comment (utay) [inline markdown/notifications.md]: `
{
  type: "deployment",
  data: {
    status: "Success",
    pusher: "utay",
    app_name: "myapp",
    app_url: "myapp.scalingo.io",
    text: "Deployment of myapp pushed by 'utay' has been successful.\nGit SHA: #12345678"
  }
}
`
Something like that is ok?

Comment (utay) [inline markdown/notifications.md]: https://github.com/Scalingo/api/pull/407

Comment (Soulou) [inline markdown/notifications.md]: Nop not exactly, we already have events for the apps

http://developers.scalingo.com/events.html

Just take the event of the deployment: https://github.com/Scalingo/api/blob/master/app/models/deployment.rb#L38

Comment (utay) [inline markdown/notifications.md]: https://github.com/Scalingo/api/pull/407/commits/f639288cb07fe5563fecee0e2615bc9f55261b24
Is that ok? Here I just take the event but I can't add additional data if needed.

Comment (utay) [inline markdown/notifications.md]: https://github.com/Scalingo/api/pull/407/commits/bcaac99f8ede2d719d811787b50af47159d8e86e
Better here.

**Comment by Soulou** [comment]
Rebase required

**Comment by utay** [comment]
Rebase done, why this branch has still conflicts?

**Comment by Soulou** [comment]
You need to rebase on  Scalingo/gh-pages, is that what you've done?

```
git pull --rebase upstream gh-pages
```

**Comment by Soulou** [comment]
BTW, this command is exactly like:

```
git fetch upstream
git rebase upstream/gh-pages
```

**Pull request closed without merging**